packages/opencensus-instrumentation-mongodb/test/test-mongodb.ts

```diff
@@ -0,0 +1,280 @@
+/**
+ * Copyright 2018, OpenCensus Authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import {types} from '@opencensus/opencensus-core';
+import {classes} from '@opencensus/opencensus-core';
+import {logger} from '@opencensus/opencensus-core';
+import * as assert from 'assert';
+import {accessSync} from 'fs';
+import * as http from 'http';
+import * as mocha from 'mocha';
+import * as mongodb from 'mongodb';
+import * as semver from 'semver';
+
+import {plugin} from '../src/';
+import {MongoDBPlugin} from '../src/';
+
+type MongoDBAccess = {
+  client: mongodb.MongoClient,
+  collection: mongodb.Collection
+};
+
+class RootSpanVerifier implements types.SpanEventListener {
+  endedRootSpans: types.RootSpan[] = [];
+
+  onStartSpan(span: types.RootSpan): void {}
+  onEndSpan(root: types.RootSpan) {
+    this.endedRootSpans.push(root);
+  }
+}
+
+function accessCollection(url: string, dbName: string, collectionName: string) {
+  return new Promise((resolve, reject) => {
+    mongodb.MongoClient.connect(url, function connectedClient(err, client) {
+      if (err) {
+        reject(err);
+        return;
+      }
+      const db = client.db(dbName);
+      const collection = db.collection(collectionName);
+      resolve({'client': client, 'collection': collection} as MongoDBAccess);
+    });
+  });
+}
+
+function assertSpan(
+    rootSpanVerifier: RootSpanVerifier, expectedName: string,
+    expectedType: string) {
+  assert.strictEqual(rootSpanVerifier.endedRootSpans.length, 1);
+  assert.strictEqual(rootSpanVerifier.endedRootSpans[0].spans.length, 1);
+  assert.strictEqual(
+      rootSpanVerifier.endedRootSpans[0].spans[0].name, expectedName);
+  assert.strictEqual(
+      rootSpanVerifier.endedRootSpans[0].spans[0].type, expectedType);
+}
+
+describe('MongoDBPlugin', () => {
+  // For these tests, mongo must be runing. Add OPENCENSUS_MONGODB_TESTS to run
+  // these tests.
+  const OPENCENSUS_NETWORK_TESTS =
+      process.env.OPENCENSUS_NETWORK_TESTS as string;
+  let shouldTest = true;
+  if (!OPENCENSUS_NETWORK_TESTS) {
+    console.log('Skipping test-mongodb. Run MongoDB to test');
+    shouldTest = false;
+  }
+
+  const url = 'mongodb://localhost:27017';
```

Comment:
Since they are constants, can all of these be capitalized?

Reply:
Sure, updated.